core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -1,7 +1,7 @@
 <!--
 The MIT License
 
-Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi,
+Copyright (c) 2004-2015, Sun Microsystems, Inc., Kohsuke Kawaguchi,
```

Comment:
Second line lists me :-D

Sent from my iPhone

> On 25 Mar 2015, at 13:08, James Nord <notifications@github.com> wrote:
> 
> In core/src/main/resources/lib/hudson/executors.jelly:
> 
> > @@ -1,7 +1,7 @@
> >  <!--
> >  The MIT License
> >  
> > -Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi,
> > +Copyright (c) 2004-2015, Sun Microsystems, Inc., Kohsuke Kawaguchi,
> I highly doubt that this is copyright in this range by Sun Microsystems.
> If you feel like modifying this then add a new line with some new dates and the new copyright holder.
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> -- 
> You received this message because you are subscribed to the Google Groups "engineering-code-reviews" group.
> To unsubscribe from this group and stop receiving emails from it, send an email to engineering-code-reviews+unsubscribe@cloudbees.com.
> To post to this group, send email to engineering-code-reviews@cloudbees.com.
> To view this discussion on the web visit https://groups.google.com/a/cloudbees.com/d/msgid/engineering-code-reviews/jenkinsci/jenkins/pull/1596/r27117918%40github.com.